### ISSUE TITLE
Contents not clearing when using CMD+L or clear when in SSH

### ISSUE THREAD

**Issue by theluk** [zeit/hyper#2371]
When in ssh, CMD+L or `clear` do not clear the contents.
Instead the cursor jumps to the top most position as expected, but all previous contents are still visible.

I mean that´s an old issue, when is this finally going to be fixed. That issue makes Hyper unusable, as for me, I use only ssh. 

        -
        Hyper 1.4.8
        Electron 1.7.8
        win32 x64 10.0.15063

**Comment by basedjux** [comment]
When you say `CMD+L` to you mean `Ctrl`?
Seeing as how your platform is marked as `win32`

**Comment by basedjux** [comment]
Just tested this.

```
Hyper 1.4.8 (stable)
Electron 1.7.8
win32 x64 10.0.15063
```
This affects all text in the CLI.
Performing commands to fill the terminal with text and then performing `clear` will move the cursor and prompt line to row 0 but will NOT clear the terminal of text.


I'm currently investigating this.

**Comment by ugurozturk** [comment]
'clear' is not recognized as an internal or external command,
operable program or batch file.

it must clear the text i guess

**Comment by basedjux** [comment]
Yes @ugurozturk, `clear` is not a valid `cmd.exe` command, it's `cls` (clear screen).

**Comment by bennyman123abc** [comment]
Is this still being looked into? I'm having this same issue on the most recent (stable) build of Hyper.
